change/@microsoft-teams-js-87caf6f6-44c9-4ff7-9cd4-e4a4c67e0ee9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "\"Added in `Env` variable expansion to `NPM_TOKEN` variable in add-npmrc powershell script\"",
```

Comment:
The escaped quotes look odd. Are they necessary?

Reply:
Yeah, I agree, I understand that the backticks can't be used since that would cancel out those around 'env' and 'npm_token' but could single quotes be used instead?

Reply:
maybe they can't tho